service/history/events/cache.go

```diff
@@ -18,54 +18,55 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-//go:generate mockgen -copyright_file ../../LICENSE -package $GOPACKAGE -source $GOFILE -destination eventsCache_mock.go
+//go:generate mockgen -copyright_file ../../../LICENSE -package $GOPACKAGE -source $GOFILE -destination cache_mock.go
 
-package history
+package events
 
 import (
 	"time"
 
 	"github.com/uber/cadence/.gen/go/shared"
-	"github.com/uber/cadence/common"
 	"github.com/uber/cadence/common/cache"
 	"github.com/uber/cadence/common/log"
 	"github.com/uber/cadence/common/log/tag"
 	"github.com/uber/cadence/common/metrics"
 	"github.com/uber/cadence/common/persistence"
+	"github.com/uber/cadence/service/history/config"
 )
 
 type (
-	eventsCache interface {
-		getEvent(
+	// Cache caches workflow history event
+	Cache interface {
+		GetEvent(
```

Comment:
comment on the public method?

Reply:
Sure I can add comments if you have strong opinion on this. But I think in this case the method name is self-explanatory so even if I add some comments it will just be some dummy words. Plus the linter doesn't complain about this. More generally, I don't think our code base has comments for all exported method, for example `ReplicationTaskFetcher`, `ShardContext`, etc.

But let me know if you still think it's necessary, I will add them.

Reply:
As long as the lint check doesn't complain, then it is fine.